### PR TITLE
Feature tests and docs

### PR DESCRIPTION
**Pull request by avoevodin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,18 +46,36 @@ postgres:13.2-alpine
 * Compile messages:
 ```shell
 cd brigadier
-./manage.py compilemessages
+./manage.py compilemessages --no-input
```
Comment (iliadmitriev) [inline README.md:49]: there is no such option --no-input in compilemessages command

File: README.md [hunk 2]
```diff
@@ -42,18 +46,36 @@ postgres:13.2-alpine
...
 ```shell
 ./manage.py runserver 0:8000
 ```
+* Create objects from admin console.
+    1. Go to the browser and type '127.0.0.1:8000/admin'
+    2. Login
+    3. Create objects
+* Run tests with coverage: 
+```shell
+coverage run --source='.' manage.py test
```
Comment (iliadmitriev) [inline README.md:74]: could be more verbose, i.e. `-v 2`

**Review by iliadmitriev — APPROVED** [review]
forget collecting static files
I approve with condition to fix this before merging

**Comment by avoevodin** [comment]
static will be done in the other branch as we've approved.

**Pull request merged**